apps/homepage/0.10.3/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 services:
   homepage:
     container_name: ${CONTAINER_NAME}
-    image: ghcr.io/gethomepage/homepage:v0.10.3
+    image: ghcr.io/gethomepage/homepage:v0.10.4
     restart: always
     networks:
       - 1panel-network
```

Comment:
The patch changes the `image` property to a newer version (`v0.10.4`) of the HomePage service from `ghcr.io/gethomepage/homepage:v0.10.3`. This is generally a good practice for updating software installations to receive the latest security patches and improvements.

There are no specific irregularities or potential issues in this change. If you plan to deploy this update, make sure that other parts of your application configuration (like environment variables) continue to work with v0.10.4 without breaking anything.